css/css-gcpm/running-007.xht

```diff
@@ -0,0 +1,26 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <title>CSS Test: content moved with running() may float float</title>
```

Comment:
Should this content actually float? According to CSS-Pages-3 only certain properties from CSS 2.1 apply in the margin context, and the float property is not one of them. 

I ended up filing an [issue against GCPM](https://github.com/w3c/csswg-drafts/issues/4788) that may influence many of these tests.

Reply:
Thanks for looking at these. I've presumed that [list of properties](https://drafts.csswg.org/css-page-3/#properties-list) were applicable to the @top-center (etc.) rules, not running content. It doesn't include "display", so if we disallowed float based on that list, we'd also be disallowing tables in running content, or `display: math`.